packages/crypto/package.json

```
@@ -0,0 +1,83 @@
+{
+  "name": "@cardano-sdk/crypto",
+  "version": "0.1.0",
+  "description": "Cryptographic types and functions for Cardano",
+  "engines": {
+    "node": ">=14.20.1"
+  },
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js"
+    }
+  },
+  "repository": "https://github.com/input-output-hk/cardano-js-sdk",
+  "publishConfig": {
+    "access": "public"
+  },
+  "sideEffects": false,
+  "contributors": [
+    "Rhys Bartels-Waller <rhys.bartelswaller@iohk.io> (https://iohk.io)",
+    "Martynas Kazlauskas <martynas.kazlauskas@iohk.io> (https://iohk.io)",
+    "Daniele Ricci <daniele.ricci@iohk.io> (https://iohk.io)",
+    "Ivaylo Andonov <ivaylo.andonov@iohk.io> (https://iohk.io)",
+    "Mircea Hasegan <mircea.hasegan@iohk.io> (https://iohk.io)",
+    "Angel Castillo Bacigalupi <angel.castillo@iohk.io> (https://iohk.io)",
+    "Dmytro Iakymenko <dmytro.iakymenko@iohk.io> (https://iohk.io)"
+  ],
+  "license": "Apache-2.0",
+  "scripts": {
+    "build:esm": "tsc -p src/tsconfig.json --outDir ./dist/esm --module es2020",
+    "build:cjs": "tsc --build src",
+    "build": "run-s build:cjs build:esm module-fixup",
+    "circular-deps:check": "madge --circular dist",
+    "module-fixup": "shx cp ../../build/cjs-package.json ./dist/cjs/package.json && cp ../../build/esm-package.json ./dist/esm/package.json",
+    "tscNoEmit": "shx echo typescript --noEmit command not implemented yet",
+    "cleanup:dist": "shx rm -rf dist",
+    "cleanup:nm": "shx rm -rf node_modules",
+    "cleanup": "run-s cleanup:dist cleanup:nm",
+    "lint": "eslint -c ../../complete.eslintrc.js \"src/**/*.ts\" \"test/**/*.ts\"",
+    "lint:fix": "yarn lint --fix",
+    "test": "jest -c ./jest.config.js",
+    "test:build:verify": "tsc --build ./test",
+    "test:e2e": "shx echo 'test:e2e' command not implemented yet",
+    "coverage": "shx echo No coverage report for this package",
+    "prepack": "yarn build"
+  },
+  "devDependencies": {
+    "@cardano-ogmios/schema": "5.5.7",
+    "@dcspark/cardano-multiplatform-lib-nodejs": "^3.1.1",
+    "@types/lodash": "^4.14.182",
+    "eslint": "^7.32.0",
+    "jest": "^28.1.3",
+    "madge": "^5.0.1",
+    "npm-run-all": "^4.1.5",
+    "shx": "^0.3.3",
+    "ts-jest": "^28.0.7",
+    "typescript": "^4.7.4"
+  },
+  "peerDependencies": {
+    "@dcspark/cardano-multiplatform-lib-nodejs": "^3.1.1"
+  },
```

Users can use any of those:
```suggestion
  "peerDependencies": {
    "@dcspark/cardano-multiplatform-lib-nodejs": "^3.1.1",
    "@dcspark/cardano-multiplatform-lib-browser": "^3.1.1",
    "@dcspark/cardano-multiplatform-lib-asmjs": "^3.1.1"
  },
  "peerDependenciesMeta": {
    "@dcspark/cardano-multiplatform-lib-nodejs": {
      "optional": true
    },
    "@dcspark/cardano-multiplatform-lib-browser": {
      "optional": true
    },
    "@dcspark/cardano-multiplatform-lib-asmjs": {
      "optional": true
    }
  },
```